netbench/cdk/logParser.py

```diff
@@ -0,0 +1,18 @@
+import sys
+
+def main(file_name):
+    f = open(file_name, "r+")
+    lines = []
+    for line in f.readlines():
+        try:
+            lines.append(line[line.index('{'):])
+        except:
+            continue
+
+    f.truncate(0)
+    f.seek(0)
+    f.writelines(lines)
+    f.close()
+
+if __name__ == "__main__":
+    main(sys.argv[1])
```

Comment:
This whole file could be replaced with an awk script:

```sh
awk -F'{' '{ st = index($0,"{");print "{" substr($0,st+1)}'
```

My bet is it's a lot faster than the python version too

demo:

```sh
$ echo 'bunch of stuff before {"json":true}\ntest foo {}' | awk -F'{' '{ st = index($0,"{");print "{" substr($0,st+1)}'
{"json":true}
{}
```

Reply:
I'll look into it, thanks for the suggestion!